examples/deprecated/centos-7.yaml

```diff
@@ -0,0 +1,22 @@
+# # This example requires Lima v0.10.0 or later.
+# WARNING: RHEL-like below version 8 is unsupported, use at your own risk!
+images:
+- location: "https://cloud.centos.org/altarch/7/images/CentOS-7-x86_64-GenericCloud-2009.qcow2"
+  arch: "x86_64"
+  digest: "sha256:e38bab0475cc6d004d2e17015969c659e5a308111851b0e2715e84646035bdd3"
+- location: "https://cloud.centos.org/altarch/7/images/CentOS-7-aarch64-GenericCloud-2009.qcow2"
+  arch: "aarch64"
+  digest: "sha256:51c0222aa4bc7d966fc044eb6ce9182993a1dc398eaa595e58abd0d361439baf"
+containerd:
+  system: false
+  user: false
```

Comment:
Please explain the reason. IIRC, CentOS 7 lacked user-scoped systemd?

Reply:
It has. Let me remove this and test just to be sure.

Reply:
You're right. I've just tried use nerdctl in the instance and got a `containerd-rootless-setuptool.sh install`, but CentOS7 lack of support to `systemctl --user`. Using `containerd:system` as true works, but only if you use as root. I will comment in the file

Reply:
Could you check again?